src/main/container/Sidebar.jsx

```diff
@@ -0,0 +1,92 @@
+import { Link } from "react-router-dom";
+import { FaHome } from "react-icons/fa";
+import { useState } from "react";
+
+export const Sidebar = () => {
+  const [isTransitionActive, setIsTransitionActive] = useState(false);
+
+  return (
+    <>
+      <div className="w-full">
+        <ul className="w-full flex items-center justify-between p-2 md:flex-col ">
+          <li className="w-1/5 md:w-full md:mb-4">
```

Comment:
make as a reusable component

Reply:
eg:

 const listItems = [{
icon: <IconName>,
label: 'Label Text',
action: () => {},
}
]

Reply:
**created list items and made it reusable**

![image](https://github.com/Anuraj-TR/spendlux/assets/101694416/123a802f-3208-4abc-9406-abbeb42743f3)


************************

![image](https://github.com/Anuraj-TR/spendlux/assets/101694416/2530ee8d-5cb4-467d-bee8-df54a618e8f5)

_**!need help with adding active state on menu items**_